packages/experimental/Menu/src/MenuList/MenuList.tsx

```diff
@@ -0,0 +1,24 @@
+/** @jsx withSlots */
+import React from 'react';
+import { View } from 'react-native';
+import { compose, UseSlots, withSlots } from '@fluentui-react-native/framework';
+import { menuListName, MenuListProps, MenuListType } from './MenuList.types';
+import { stylingSettings } from './MenuList.styling';
+
+export const MenuList = compose<MenuListType>({
+  displayName: menuListName,
+  ...stylingSettings,
+  slots: {
+    root: View,
+  },
+  useRender: (userProps: MenuListProps, useSlots: UseSlots<MenuListType>) => {
+    const Slots = useSlots(userProps);
+
+    return (_final: MenuListProps, children: React.ReactNode) => {
+      // TODO add gap between children
```

Comment:
Going to implement this in separate PR since this is already pretty big

Reply:
The "Stack" component I think exists to handle gap?

Reply:
Oh that's true, I could just use Stack instead of reimplementing it. Hmm.

Reply:
Can still be done separately, just pointing it out as an initial venue to explore =)

Reply:
Removing gap for now.